### PR TITLE
Removed ResourceExposure/ResourceConsumption attributes

### PR DESCRIPTION
**Pull request by zacpwhite**
Remove the obsolete ResourceExposure / ResourceConsumption attributes as outlined in #321

### CODE REVIEW

File: src/System.Windows.Forms.Design.Editors/src/Misc/NativeMethods.cs [hunk 1]
```diff
@@ -3901,113 +3898,86 @@ public const int
...
 
         [DllImport(ExternDll.User32, ExactSpelling = true, CharSet = CharSet.Auto)]
-        [ResourceExposure(ResourceScope.None)]
         public static extern bool GetWindowRect(IntPtr hWnd, [In] [Out] ref RECT rect);
 
         [DllImport(ExternDll.Ole32, ExactSpelling = true, CharSet = CharSet.Auto)]
-        [ResourceExposure(ResourceScope.None)]
         public static extern int RevokeDragDrop(IntPtr hwnd);
 
-        [ResourceExposure(ResourceScope.None)]
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Misc/NativeMethods.cs]: nit: some of these changes remove the attribute declaration as well as the rest of the line, while others keep the line. Could we choose either and be consistent? I think deleting the line would be a little cleaner.

File: src/System.Windows.Forms.Design.Editors/src/Misc/UnsafeNativeMethods.cs [hunk 2]
```diff
@@ -14,95 +14,95 @@ namespace System.Windows.Forms.Design
     internal class UnsafeNativeMethods
     {
         [DllImport(ExternDll.User32, ExactSpelling = true, CharSet = CharSet.Auto)]
-        [ResourceExposure(ResourceScope.None)]
+
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Misc/UnsafeNativeMethods.cs]: Here the line was kept.

**Comment by zsd4yr** [comment]
thank you @zacpwhite

**Comment by zacpwhite** [comment]
No problem, although I was in the process of cleaning up even more formatting issues as a result of this removal. I'll submit another PR in the future to clean the rest of these up!

**Comment by RussKie** [comment]
Please use GH special keywords like "Fixes" or "Closes" to have the underlying issue automatically closed upon merge.
#321 should be closed now.

**Pull request merged**